transmogrifier/sources/xmltransformer.py

```diff
@@ -44,46 +42,6 @@ def parse_source_file(cls, source_file: str) -> Iterator[Tag]:
                 yield record
                 element.clear()
 
-    @final
-    def transform(self, source_record: Tag) -> timdex.TimdexRecord | None:
-        """
-        Call Transformer._transform method to transform XML record to TIMDEX record.
-
-        May not be overridden.
-
-        Args:
-            source_record: A BeautifulSoup Tag representing a single XML record.
-        """
-        return self._transform(source_record)
```

Comment:
Nice work on this removal, and subsuming this into `Transformer.transform()` method!  Same for `JSONTransformer`.